include/dfu/dfu_context_handler.h

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/** @file
+ * @brief header.
+ */
+
+#ifndef DFU_CONTEXT_HANDLER_H__
+#define DFU_CONTEXT_HANDLER_H__
+
+/**
+ * @defgroup event_manager Event Manager
+ * @brief Event Manager
+ *
+ * @{
+ */
+
+#include <zephyr/types.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @brief Interface which needs to be supported by all DFU contexts.
```

Comment:
You call this an interface here, but the type is named context..

Reply:
I agree it's a bit confusing. The intent was to communicate that the `dfu_ctx_handler` has this interface. As `interface` is not a supported construct in `C`. If we are changing the naming to dfu and calling it `dfu_interface` may help a bit in the confusion do you agree with this change?

Reply:
I propose that we change the type name to `dfu_ctx_interface`

Reply:
Renamed "context" and "ctx" to "target"